src/pages/index.tsx

```diff
@@ -46,6 +47,9 @@ export const Home = () => {
                   </SwiperSlide>
                 ))}
               </MainSlider>
+
+              <TypographyCustom tag="h2" typographyText="Більше товарів для вибору" />
+              <TypographyCustom tag="h1" typographyText="Ноутбуки та комп'ютери" />
```

Comment:
what if you need styles for the h1 tag but the actual tag to render you need h2?

Please consider using `variant` props and `component` or `tag` to render the proper tag.

Reference: https://mui.com/material-ui/react-typography/